novawallet/Common/Protocols/FeatureSupportChecking.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+protocol FeatureSupportChecking {
+    func checkingSupport(
+        of feature: SupportCheckingFeatureType,
+        for metaAccountModel: MetaAccountModel,
+        sheetPresentingView: ControllerBackedProtocol?,
+        successClosure: @escaping () -> Void
+    )
+}
+
+// MARK: - Multisig check
+
+private extension FeatureSupportChecking where Self: MessageSheetPresentable {
+    func checkMultisigSellSupport(
+        for multisigMetaAccount: MetaAccountModel,
+        sheetPresentingView: ControllerBackedProtocol,
+        successClosure: @escaping () -> Void
+    ) {
+        let unsupportedClosure: () -> Void = { [weak self] in
+            self?.presentFeatureUnsupportedView(
+                from: sheetPresentingView,
+                type: .sell,
+                walletType: .init(walletType: multisigMetaAccount.type),
+                completion: {}
+            )
+        }
+
+        checkIfMultisigThreshold1(
```

Comment:
This approach will not work for complex account structure - when current wallet is proxy and there is a multisig somewhere in hierarchy. We need integrate `WalletDelayedExecVerifier` for that.